src/main/resources/db/migration/common/V1_352__add_eod_manifest_file_index.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS eod_manifest_file_idx ON external_object_directory (manifest_file)
```

Comment:
i've a funny feeling that postgres didn't use the standard index with a % at teh end, that it needed to specify the type, let me double check this by looking at the explain plan

Reply:
ok cheers. According to documentation it should use the standard index (B-tree) for LIKE 'DARTS%'. 
If that's not the case then there's something weird going on with our Postgres

Reply:
as discussed, we need the `text_pattern_ops` as part of the index

```sql
CREATE INDEX IF NOT EXISTS eod_manifest_file_idx ON external_object_directory (manifest_file text_pattern_ops);```

Reply:
nice spot thanks